data-prepper-plugins/sns-sink/src/main/java/org/opensearch/dataprepper/plugins/sink/SNSSinkConfig.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.dataprepper.plugins.sink;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import org.opensearch.dataprepper.model.configuration.PluginModel;
+import org.opensearch.dataprepper.plugins.sink.configuration.AwsAuthenticationOptions;
+import org.opensearch.dataprepper.plugins.sink.configuration.BufferTypeOptions;
+import org.opensearch.dataprepper.plugins.sink.configuration.ThresholdOptions;
+
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * sns sink configuration class contains properties, used to read yaml configuration.
+ */
+public class SNSSinkConfig {
+
+    private static final int DEFAULT_CONNECTION_RETRIES = 5;
+    private static final int DEFAULT_UPLOAD_RETRIES = 5;
+    public static final String STS_REGION = "sts_region";
```

Comment:
This is just `region` in other places in data prepper

Reply:
Thanks @graytaylor0 for your feedback. we have incorporated the changes.